src/lib/helper.js

```diff
@@ -18,7 +18,23 @@ const getItemsNamesFromDoc = (items) => {
   }, []);
 };
 
-const fromMilleToHours = (time) =>
-  (Date.now() - new Date(time)) / (1000 * 60 * 60);
+const getUTCNowInMilliSec = () => {
+  const now = new Date();
+  // get time now in milliseconds
+  const time = now.getTime();
+  // get timezone offset in minutes and convert it to milliseconds
+  // offset is how many milliseconds you should add to your time to convert to UTC
+  const offset = now.getTimezoneOffset() * 60000;
+  return time + offset;
+};
+
+const fromMilliSecToHours = (time) => time / (1000 * 60 * 60);
 
-export { existingName, fromMilleToHours };
+const fromMilliSecToDays = (time) => Math.floor(time / (1000 * 60 * 60 * 24));
```

Comment:
I’m curious what made you choose `floor` and not `round` or `ceil`?

Reply:
Well no particular reason tbh, I guess we just used the method that came to mind most readily😅

Reply:
After some discussions, we decided that `Math.ceil` makes more sense
Because if we supposed it's the first day in the world of JS and we passed 0.3 of it 
then using `floor` and `round` will make it day 0 and using `ceil` will make it day 1
and I think it should be counted as day 1
Thanks a lot for the valuable comments Andrew 💜